clang/test/CIR/CodeGen/union-padding.c

```diff
@@ -19,7 +19,7 @@ short use() {
 // CHECK:       !rec_anon_struct = !cir.record<struct  {!s16i, !cir.array<!u8i x 2>}>
 
 // CHECK:       @g3 = #cir.global_view<@g2> : !cir.ptr<!cir.ptr<!s16i>>
-// CHECK:       @g2 = #cir.const_array<[#cir.global_view<@g1, [1]> : !cir.ptr<!s16i>]> : !cir.array<!cir.ptr<!s16i> x 1>
+// CHECK:       @g2 = #cir.const_array<[#cir.global_view<@g1, offset 4> : !cir.ptr<!s16i>]> : !cir.array<!cir.ptr<!s16i> x 1>
```

Comment:
Would be good to check some LLVM output in this test as an overall thing. Is the `offset 4` translates to `4` in LLVM as well?

Reply:
I will do it!